README.md

```diff
@@ -66,14 +66,13 @@ pip install scikit-learn
 Run the following code:
 
 ```python
-    
+import numpy as np
 from sklearn.datasets import fetch_20newsgroups
 from sklearn.model_selection import StratifiedKFold
 from sklearn.pipeline import Pipeline
 from sklearn.feature_extraction.text import TfidfVectorizer
 from sklearn.linear_model import LogisticRegression
-
-from ml_report import MLReport
+from ml_report_kit import MLReport
```

Comment:
Thanks for fixing it 👍🏽 

You can leave the space between the groups of imports. It's part of PEP8 - grouping of imports, see: 

[https://peps.python.org/pep-0008/#imports](https://peps.python.org/pep-0008/#imports)

> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> 
> You should put a blank line between each group of imports.